.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: node_js
 node_js:
   - "6.10.3"
+install: npm install
```

Comment:
is there a reason for adding npm install?, I think travis runs `npm install` for node apps automatically.